pkg/collector/corechecks/ebpf/probe/ebpfcheck/probe.go

```diff
@@ -270,11 +283,15 @@ func (k *Probe) getProgramStats(stats *model.EBPFStats) error {
 			RunCount:        info.RunCnt,
 			RecursionMisses: info.RecursionMisses,
 		}
+		key := progKey(ps)
+		if _, ok := uniquePrograms[key]; ok {
+			continue
+		}
```

Comment:
This will skip all other instances of the program?

As I understand this problem only happens with tc classifiers. Maybe skipping that class altogether is a better option, rather than emitting incomplete information. Especially because it will not be obvious in the check that this is being done.

Reply:
It only skips programs that are loaded multiple times with the same name+type+module. AFAICT this is limited to TC classifiers at the moment.

De-duplicating names in the way I was before increases the cardinality, and there is no determinism so you don't really know which one is `_2` or `_3`, etc.

Reply:
right now just ignores, but we could explore averaging or something in the future.